File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l,r+1):
        check = 1
        for s in str(i):
            if s == '5' or s== '0':
                pass
            else:
                check = 0
        if check == 1:
            answer.append(i)
    if len(answer) == 0:
        answer.append(-1)
    return answer

File: 프로그래머스/unrated/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    let result = [];
    for(let i=l;i<=r;i++){
        let num = i.toString();
        let check = true;
        let word= '';
        for(char of num){
            if(char==='5' || char==='0'){
                word+=char;
            }else{
                check = false;
            }
        }
        if(check){
            result.push(i);
        }
    }
    if(result.length>0){
        return result;
    }else{
        return [-1];
    }
}